### DISCUSSION TITLE
Add proxy and run more than one bot per machine

### DISCUSSION THREAD

**Discussion by WrightWebsites** [mifi/instauto#135] [Ideas]
How would one go about adding a proxy variable in the example.js so that you could run the script via a 4G or residential proxy.

**Comment by WrightWebsites** [comment]
Going one step further when the script is run it would create a folder in a folder "database" (for example) which would within have the Instagram username as a folder and then within that save the json files: liked media, followed, unfollowed and cookies etc.

**Comment by mifi** [comment]
See #31 and https://stackoverflow.com/questions/52777757/how-to-use-proxy-in-puppeteer-and-headless-chrome

**Reply by iamlwright** [reply]
That is awesome thank you, sorry I missed the question had already been asked.

**Reply by WrightWebsites** [reply]
Do you have an example of it in use you could share? Struggling to get around a "only accepts http error"

**Reply by mifi** [reply]
unfortunately not but if someone finds out how, feel free to submit a PR for the README.md

**Reply by mifi** [reply]
i added an example here:
https://github.com/mifi/instauto/blob/master/example.js
dunno if it works tho
